Waste.API/Controllers/LookupController.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.AspNetCore.Mvc;
+using Waste.API.Models;
+using WasteManagement.API.Data;
+
+namespace WasteManagement.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class LookupController : ControllerBase
+    {
+        private readonly ILogger<LookupController> _logger;
+
+        public LookupController(ILogger<LookupController> logger)
+        {
+            _logger = logger;
+        }
+
+        [HttpGet(Name = "GetLookupData")]
+        public IEnumerable<LookupData> Get()
+        {
+            List<LookupData> data = new List<LookupData>();
+            
+            foreach (var item in ReadWasteTypes())
+            {
+                LookupData lookupData = new LookupData();
+                lookupData.Id = item.Id;
+                lookupData.Name = item.Name;
+                data.Add(lookupData);
+            }
```

Comment:
AutoMapper is being used in the project. Please use it to map the types.

Reply:
This code is just for POC work, will probably change?

Reply:
Removed the class.